apis/server/image_bridge.go

```diff
@@ -131,7 +131,6 @@ func (s *Server) removeImage(ctx context.Context, rw http.ResponseWriter, req *h
 
 	label := util_metrics.ActionDeleteLabel
 	defer func(start time.Time) {
-		metrics.ImageActionsCounter.WithLabelValues(label).Inc()
```

Comment:
maybe we could judge the return err here to decide how to handle the counter? if err was not found, not add it?

Reply:
it's more clear I think